trie/sync.go

```diff
@@ -292,8 +295,11 @@ func (s *Sync) children(req *request, object node) ([]*request, error) {
 			if _, ok := s.membatch.batch[hash]; ok {
 				continue
 			}
-			if ok, _ := s.database.Has(node); ok {
-				continue
+			if s.bloom.Contains(node) {
+				// Bloom filter says this might be a duplicate, double check
+				if ok, _ := s.database.Has(node); ok {
+					continue
+				}
```

Comment:
At this point, I'd like an else-block we note that the bloom delivered a FP, with a metric. That way, we can track bloom saturation and see how it degrades.

Reply:
Ah, I see, you're calculating it theoretically instead. Yes, I guess that works fine too

Reply:
done